src/main/java/io/github/opencubicchunks/cubicchunks/mixin/debug/common/MixinMinecraftServer.java

```diff
@@ -90,11 +90,11 @@ private void prepareLevels(ChunkProgressListener worldGenerationProgressListener
             this.waitUntilNextTick();
         }
 
-        try {
-            Thread.sleep(10000);
-        } catch (InterruptedException e) {
-            int ignored = 0;
-        }
+//        try {
+//            Thread.sleep(10000);
+//        } catch (InterruptedException e) {
+//            int ignored = 0;
+//        }
```

Comment:
This should not be commented out. It should be disabled through the mixin.

Reply:
if the `cubicchunks.debug` jvm arg is not set this mixin will never apply anyway.
This was here to wait on loading into the world to make sure all of the cubes were properly generated